css-syntax/styles.css

```diff
@@ -0,0 +1,28 @@
+body {
+color: #090302;
+font-family: sans-serif
+}
+h1 {
+text-align: center;
+}
+article {
+  padding-left: 16px;
+  padding-right: 16px;
+  border: 1px solid;
+  border-radius: 6px;
+  margin-bottom: 16px;
+}
+.title{
+  font-family: monospace;
+  font-size: 24px;
+}
+
+ #html , h2{
```

Comment:
This is selecting any element with an `id` attribute set to `html`, but it is also styling all `h2` elements, because you have given it as a separate selector using a comma. You can remove the `h2` portion from this selector.

Please also fix this for the below selectors selecting `id`s